server/pulp/plugins/loader/api.py

```diff
@@ -282,6 +296,19 @@ def is_valid_cataloger(cataloger_id):
 
 # plugin api -------------------------------------------------------------------
 
+def get_unit_model_by_id(model_id):
+    """
+    Get the ContentUnit model class that corresponds to the given id.
+
+    :param model_id: id of the model
+    :type model_id: str
```

Comment:
Consider a newline underneath this separating the input parameters from output parameters.